styles/components/_view.scss

```diff
@@ -1,3 +1,7 @@
+// .view {
```

Comment:
If needed, implement for menu view and mode view

Reply:
I think that we need to tweak these names of files and classes. General .view is in emptyView.scss (btw. named like this because there already was view.scss) and there .view should be styled. As this whole file concerns GameView as should be named _gameView.scss I believe.